spec/requests/workflows_spec.rb

```diff
@@ -1636,7 +1636,7 @@ def find_file_row(name)
 
         expect(page).to have_current_path("/workflows/#{@workflow.external_id}/emails")
 
-        expect(page).to have_selector("[role=alert].warning", text: "Your name contains a colon (:) which causes email delivery problems and will be removed from the sender name when emails are sent.")
+        expect_alert_message("Your name contains a colon (:) which causes email delivery problems and will be removed from the sender name when emails are sent.")
```

Comment:
Fixes related spec - https://github.com/antiwork/gumroad/actions/runs/20308195190/job/58332073440?pr=2226

The other failing one is unrelated to our changes - hub.com/antiwork/gumroad/actions/runs/20308195190/job/58332073458